### PR TITLE
taxonomy: Labels and categories additions

### PR DESCRIPTION
**Pull request by teolemon**
### What
- Labels and categories additions

### CODE REVIEW

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -73257,6 +73257,10 @@ ciqual_food_code:en:6002
 ciqual_food_name:en:Beef, shoulder, raw
 ciqual_food_name:fr:Boeuf, épaule, crue
 
+<en:Beef meat
+en:Beef liver
+fr:Foie de boeuf
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:73262]: category names should be in plural (it's an issue for many CIQUAL categories that were added as singular); the singular should be added as a synonym

**Review by stephanegigandet — APPROVED** [review]

File: taxonomies/categories.txt (hunk 1 above)
Comment (teolemon) [inline taxonomies/categories.txt:73262]:
```suggestion
<en:Beef meat
en:Beef liverw
fr:Foies de boeuf
```

**Comment by benbenben2** [comment]
@teolemon , @stephanegigandet please review applied changes

**Review by stephanegigandet — APPROVED** [review]
Looks good, thank you!

**Pull request merged**